manifests/user.pp

```diff
@@ -13,9 +13,10 @@
   }
 
   ssh_authorized_key { "${username}_default":
-    user => $username,
-    type => $pubkey_type,
-    key  => $pubkey,
+    user   => $username,
+    type   => $pubkey_type,
+    key    => $pubkey,
+    target => "${home}/.ssh/authorized_keys2",
```

Comment:
so does this cause the ssh key resource to tell ssh to use both this and authorized_keys?